src/fulltrade/FullTradeCard.js

```diff
@@ -57,11 +57,11 @@ export default class BaseTradeCard extends Component {
                     assetName={selectedSymbol}
                     onClosePanel={() => actions.removeTrade(index)}
                 />
-                {tick && <TradeChart
+                {(tick && Object.keys(tick).length > 0) && <TradeChart
```

Comment:
Its still an array. The use of Object.keys there is not necessary though not an issue :wink .